Add KUBELET and CADVISOR to monitoring components

TheSpiritXIII opened the pull request:
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
container: added `KUBELET` and `CADVISOR` options to `monitoring_config.enable_components` in `google_container_cluster` resource
```

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report

Your PR generated some diffs in downstreams - here they are.

`google` provider: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-10605-old..auto-pr-10605) ( 3 files changed, 3 insertions(+), 3 deletions(-))
`google-beta` provider: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10605-old..auto-pr-10605) ( 3 files changed, 3 insertions(+), 3 deletions(-))

modular-magician commented:
#### Tests analytics
Total tests: `192`
Passed tests: `181`
Skipped tests: `10`
Affected tests: `1`

<details><summary>Click here to see the affected service packages</summary><blockquote><ul><li>container</li></ul></blockquote></details>

#### Action taken
<details> <summary>Found 1 affected test(s) by replaying old test recordings. Starting RECORDING based on the most recent commit. Click here to see the affected tests</summary><blockquote>TestAccContainerCluster_withMonitoringConfig </blockquote></details>

[Get to know how VCR tests work](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/contributing/#general-contributing-steps)

modular-magician commented:
$\textcolor{red}{\textsf{Tests failed during RECORDING mode:}}$
`TestAccContainerCluster_withMonitoringConfig`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-10605/artifacts/bb0136e5-6379-4f5b-a73f-3dd1beb3d3aa/build-log/recording_build/TestAccContainerCluster_withMonitoringConfig_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-10605/artifacts/bb0136e5-6379-4f5b-a73f-3dd1beb3d3aa/recording/TestAccContainerCluster_withMonitoringConfig.log)]


$\textcolor{red}{\textsf{Please fix these to complete your PR.}}$
View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-10605/artifacts/bb0136e5-6379-4f5b-a73f-3dd1beb3d3aa/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-10605/artifacts/bb0136e5-6379-4f5b-a73f-3dd1beb3d3aa/recording) for each test

shuyama1 reviewed (commented):
The test failed with error `Error: googleapi: Error 400: Request contains an invalid argument., badRequest`

Do we know if these values are already supported?

TheSpiritXIII commented:
@shuyama1 thanks for the reply!

> Do we know if these values are already supported?

They're only available on [GKE 1.29 and above](https://cloud.google.com/kubernetes-engine/docs/how-to/configure-metrics#enable-cadvisor-kubelet).

I removed them from the test and added a note about support on the HTML page. Let me know if you want me to do any further changes. Thanks again!

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report

Your PR generated some diffs in downstreams - here they are.

`google` provider: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-10605-old..auto-pr-10605) ( 2 files changed, 2 insertions(+), 2 deletions(-))
`google-beta` provider: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10605-old..auto-pr-10605) ( 2 files changed, 2 insertions(+), 2 deletions(-))

modular-magician commented:
#### Tests analytics
Total tests: `193`
Passed tests: `183`
Skipped tests: `10`
Affected tests: `0`

<details><summary>Click here to see the affected service packages</summary><blockquote><ul><li>container</li></ul></blockquote></details>

$\textcolor{green}{\textsf{All tests passed!}}$
View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-10605/artifacts/8ccc1e83-ba0b-4ac3-a4c6-eb455679414f/build-log/replaying_test.log)
